### PR TITLE
New DAshboard, and layout stylings

### LINKED ISSUE #283: Implement wireframes - UX draft
┆Attachments: <a href="https:&#x2F;&#x2F;app.asana.com&#x2F;app&#x2F;asana&#x2F;-&#x2F;download_asset?asset_id&#x3D;391077821373054">Whare_Hauora_Web.pdf</a>

### PR DESCRIPTION
**Pull request by Br3nda**
New layout as per designs

Note: This is reviving an old branch. Will need lots of updating

closes #283

Testing check list
https://gist.github.com/Br3nda/3da028a94544f6fe058bf5e30548793a

### CODE REVIEW

**Comment by Br3nda** [comment]
This pull request **introduces 4 alerts** when merging b8437486d2816e0f865c74a234b5d4a7985e08bd into 0f1aabb7b8a04fd467143362ef59d636a4c24f1b - [view on LGTM.com](https://lgtm.com/projects/g/WhareHauora/wharehauora-server/rev/pr-01e3f4989a7929a7224a31917e7fd2680eb347be)

**new alerts:**

* 4 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by Br3nda** [comment]
This pull request **introduces 4 alerts** when merging 159bcb99fd05772f0550400410d8474664b488b5 into 0f1aabb7b8a04fd467143362ef59d636a4c24f1b - [view on LGTM.com](https://lgtm.com/projects/g/WhareHauora/wharehauora-server/rev/pr-799207366d1bd591392524fd6e003c90d68e9a82)

**new alerts:**

* 4 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by Br3nda** [comment]
This pull request **introduces 4 alerts** when merging 4c904b504ce80ef40e5c92fb33998f4b99af5903 into 0f1aabb7b8a04fd467143362ef59d636a4c24f1b - [view on LGTM.com](https://lgtm.com/projects/g/WhareHauora/wharehauora-server/rev/pr-29dd670871d614946123e28f06c997b4c4f5bc9a)

**new alerts:**

* 4 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by Br3nda** [comment]
This pull request **introduces 4 alerts** when merging 3d7071cb785457f18fd66a8807106702a5c5081e into 0f1aabb7b8a04fd467143362ef59d636a4c24f1b - [view on LGTM.com](https://lgtm.com/projects/g/WhareHauora/wharehauora-server/rev/pr-34a83bff088d83b9ac2f49590243d1755d763a20)

**new alerts:**

* 4 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by Br3nda** [comment]
This pull request **introduces 4 alerts** when merging 68f0565743b31778c5883848006d4b7557740b64 into 0f1aabb7b8a04fd467143362ef59d636a4c24f1b - [view on LGTM.com](https://lgtm.com/projects/g/WhareHauora/wharehauora-server/rev/pr-9c9a096fe8880c7af0acd0cd844c080c658d0290)

**new alerts:**

* 4 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
@jakebriggs if you have time, look at the pretty i made

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
Mobile /small screen needs work

File: app.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "wharehauora-server",
   "scripts": {
-    "postdeploy": "bundle exec rake db:migrate db:seed uat:testdata"
+    "postdeploy": "bundle exec rails db:migrate db:seed uat:testdata"
```
Comment (jakerabid) [inline app.json]: whats this change for? Isn't the db a rake task?

File: app/models/home_type.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class HomeType < ApplicationRecord
-  validates :name, uniqueness: true
+  validates :name, uniqueness: true, presence: true
```
Comment (jakerabid) [inline app/models/home_type.rb]: :+1:

File: app.json (hunk 1 above)
Comment (Br3nda) [reply app.json]: in rails 5 it changed to be something you ask rails to do. Both work for now.

**Pull request merged**